src/game/Game.java

```diff
@@ -30,6 +30,8 @@
 
 	private int moveTimer;
 
+	private HashSet<Coordinate> deadCoors;
```

Comment:
keeping track of the coordinates of pieces that die because we only move the images of the affacted pieces, so we have to manually tell the gui to stop showing dead pieces